source/release-notes.txt

```diff
@@ -10,6 +10,25 @@ Release Notes
    :depth: 1
    :class: twocols
 
+1.8.0 Changelog
+---------------
+
```

Comment:
(punctuation globally)
We'll want each of these list items to end with punctuation.